pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.facebook.airlift</groupId>
         <artifactId>airbase</artifactId>
-        <version>95</version>
+        <version>103</version>
```

Comment:
please remind me, why do we need a newer version of airbase

Reply:
We don't. Just seemed like an obvious thing to update. I'll get rid of it, keep the PR only about Netty.